modules/special-rules/pages/tyranid-objectives.adoc

```diff
@@ -0,0 +1,16 @@
+= Tyranid Objectives
+
+Always ready for a krumpin, Orks will often charge into melee before the enemy has even realised that the Orks are close enough to charge them!
+
+---
+
+Apply these rules to Tyranid armies:
+
+* The army does not use objectives, regardless of scenario.
+* If an enemy detachment is wiped out, add its morale value to the Tyranid player's army morale in the Rally phase. [either/or]
+* If an enemy detachment is wiped out, in the Rally phase the Tyranid player's army morale goes up by the detachment's morale value. [either/or]
+
+NOTE: If the enemy detachment is a war engine that has been destroyed -- rather than being wrecked -- this is in addition to gaining half the war engines's morale value.
```

Comment:
```suggestion
NOTE: If the enemy detachment is a war engine, and you destroyed it -- not wrecked it-- then you gain these extra army morale points *and* the usual points equivalent to half the morale value of the war engine detachment.
```
However... what should happen if the detachment has multiple war engines and only one was destroyed, maybe on an earlier turn?

Reply:
'you destroyed it' doesn't feel appropriate for a Catastrophic Damage Chart result, as that's a second-order effect rather than an action that you performed.

'points equivalent to half the morale value of the war engine detachment' is a horrible mouthful. ;)

> However... what should happen if the detachment has multiple war engines and only one was destroyed, maybe on an earlier turn?
And this is another reason why the suggested replacement doesn't work - the extra army morale for a 'destroyed' enemy war engine is per-war engine not per-detachment.

To be honest, I'm not tied to keeping the note and it's just a reminder that getting a destroyed result on a solo enemy war engine is going to give you 150% of its morale value. So we can delete it.

```suggestion

```

Reply:
I wasn't entirely happy with my formulation of the paragraph. But the point was to clarify the specific chart result as opposed to disambiguate more general use of 'destroyed'.

As to the horrible mouthful - maybe, but I'd argue that your original construction isn't wholly clear about the relationship and difference morale value with army morale. Also I sought to remove the two avoidable -ing words and replace passive case with active.

However, I agree that we're probably safe to lose the note here — so let's do that rather than labour over it. We can always return to it in future if we have second thoughts.